### PR TITLE
Master

### PR DESCRIPTION
**Pull request by aderhall**
I fixed the readme so that it doesn’t give wrong instructions anymore.

### CODE REVIEW

**Comment by ErikBoesen** [comment]
What's this "Vae Victis" user?

File: bot.py [hunk 1]
```diff
@@ -67,6 +67,7 @@
             # Start a git pull to update bot
             print(str(subprocess.Popen('git pull', shell=True, stdout=subprocess.PIPE).stdout.read()))
             await client.send_message(message.channel, 'Update Successful! Restarting...')
+            subprocess.Popen('echo ' + message.channel + ' | cat > LastId', shell=True, stdout=subprocess.PIPE).stdout.read()
             # Restart
             os.system('python3 launch.py')
```
Comment (Twinters007) [inline bot.py]: Adrian you don't have the most up to date code.

File: bot.py [hunk 2]
```diff
@@ -67,6 +67,7 @@
             # Start a git pull to update bot
             print(str(subprocess.Popen('git pull', shell=True, stdout=subprocess.PIPE).stdout.read()))
             await client.send_message(message.channel, 'Update Successful! Restarting...')
+            subprocess.Popen('echo ' + message.channel + ' | cat > LastId', shell=True, stdout=subprocess.PIPE).stdout.read()
```
Comment (Twinters007) [inline bot.py]:
```Python
with open("LastID", "w") as text_file:
    print(message.channel, file=text_file)
```
is a much better way of doing this.

File: bot.py [hunk 3]
```diff
@@ -8,7 +8,7 @@
 # Initialize bot client
 # TODO: Make bot a class like normal bots.
 client = discord.Client()
-
+lastId = subprocess.Popen('cat LastId', shell=True, stdout=subprocess.PIPE).stdout.read()
```
Comment (Twinters007) [inline bot.py]:
```Python
with open ("LastID") as text_file:
    global lastID = text_file.read()
```
@virtuald is it "Pythonic" to make global variables like this?

Comment (virtuald) [inline bot.py]: Generally, I try to avoid globals.

File: bot.py (hunk 2 above)
Comment (ErikBoesen) [inline bot.py]: Adrian: I know you like bash (I do too), and I understand it's easier to do it in this way, but you should avoid using subprocesses. There are very, _very_ few occasions where using a bash command is the best way to do things. It's best to look up a native Python (or your current language) method. This way, your application will be _significantly_ faster, lighter, and cross-platform.

(And yes I know you corrected it here. Just a tip for the future.)

**Pull request merged**